### PR TITLE
Add ESC key and set_escdelay

### PR DESCRIPTION
**Pull request by ywkw1717**

### CODE REVIEW

File: ncurses.go [hunk 1]
```diff
@@ -253,6 +253,11 @@ func ResizeTerm(nlines, ncols int) error {
 	return nil
 }
 
+// Set the ESCDELAY
+func SetEscDelay(size int) {
```
Comment (rthornton128) [inline ncurses.go]: I don't have a machine available for testing right now but it looks like this function is an ncurses only extension and is not available in PDCurses. In order to accept this patch, I would need a NOOP wrapper. For examples see: https://github.com/rthornton128/goncurses/blob/master/goncurses.c

Don't forget to add a header to goncurses.h and use `goncurses_set_escdelay()` here.

File: ncurses.go [hunk 2]
```diff
@@ -253,6 +253,11 @@ func ResizeTerm(nlines, ncols int) error {
 	return nil
 }
 
+// Set the ESCDELAY
```
Comment (rthornton128) [inline ncurses.go]: A slightly more detailed description would be helpful. This is an area of goncurses that could definitely be improved on. No need to go into much detail or do a copy and paste. Just slightly more detail.

Example: `Sets the delay from when the escape key is pressed until ...`

File: goncurses.c [hunk 3]
```diff
@@ -88,3 +88,13 @@ int ncurses_untouchwin(WINDOW *win) { return untouchwin(win); }
 int ncurses_wattrset(WINDOW *win, int attr) { return wattrset(win, attr); }
 int ncurses_wstandend(WINDOW *win) { return wstandend(win); }
 int ncurses_wstandout(WINDOW *win) { return wstandout(win); }
+
+bool goncurses_set_escdelay(int size) {
+  bool code = true;
```
Comment (rthornton128) [inline goncurses.c]: It's not necessary to use a temp variable for something like this. You can just `return bool` from inside the `#ifdef` block. Do the same within the `#else` block for returning true. Everything else looks good.

Comment (ywkw1717) [reply goncurses.c]: OK. Thank you.

I fixed. Please confirm.

**Pull request merged**